apps/site/lib/site_web/views/mode_view.ex

```diff
@@ -26,20 +26,6 @@ defmodule SiteWeb.ModeView do
   def get_route_group(route_type, route_groups), do: route_groups[route_type]
 
   @spec fares_note(String) :: Phoenix.HTML.safe() | String.t()
-  @doc "Returns a note describing fares for the given mode"
-  def fares_note("Commuter Rail") do
-    content_tag :p do
-      [
-        "Fares for the Commuter Rail are separated into zones that depend on your origin and destination. Find your fare cost by entering your origin and destination
-      or view ",
-        link(
-          "table of fare zones.",
-          to: cms_static_page_path(SiteWeb.Endpoint, "/fares/commuter-rail-fares/zones")
-        )
-      ]
-    end
-  end
-
   def fares_note(_mode) do
     ""
   end
```

Comment:
Probably meant to delete this

Reply:
This was actually intentional, temporarily. The CMS content feeding dev-green still appears to be using this. Once we can remove that then this can go.

Reply:
Added a comment documenting this.